moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -178,9 +187,7 @@ const ProjectsListViewTable = ({ query, searchTerm }) => {
    * Query Management
    */
   // Manage the ORDER BY clause of our query
-  if (sort.column !== "" && sort.order !== "") {
-    query.setOrder(sort.column, sort.order);
-  }
+  query.setOrder(sort.column, sort.order);
```

Comment:
this setter runs every render, and is somehow not causing an infinite loop with the way it interacts with the query that `useQuery` is consuming. anyway, we didn't go to far into the code, but this hints at some underlying GQLAbstract behavior that needs a deeper look.